framework/decode/vulkan_export_json_consumer_base.cpp

```diff
@@ -621,6 +622,30 @@ void VulkanExportJsonConsumerBase::Process_vkUpdateDescriptorSetWithTemplateKHR(
     WriteBlockEnd();
 }
 
+void VulkanExportJsonConsumerBase::Process_vkCreateRayTracingPipelinesKHR(
+    const ApiCallInfo&                                               call_info,
+    VkResult                                                         returnValue,
+    format::HandleId                                                 device,
+    format::HandleId                                                 deferredOperation,
+    format::HandleId                                                 pipelineCache,
+    uint32_t                                                         createInfoCount,
+    StructPointerDecoder<Decoded_VkRayTracingPipelineCreateInfoKHR>* pCreateInfos,
+    StructPointerDecoder<Decoded_VkAllocationCallbacks>*             pAllocator,
+    HandlePointerDecoder<VkPipeline>*                                pPipelines)
+{
+    nlohmann::ordered_json& jdata = WriteApiCallStart(call_info, "vkCreateRayTracingPipelinesKHR");
+    FieldToJson(jdata[NameReturn()], returnValue, json_options_);
+    auto& args = jdata[NameArgs()];
+    HandleToJson(args["device"], device, json_options_);
+    HandleToJson(args["deferredOperation"], deferredOperation, json_options_);
+    HandleToJson(args["pipelineCache"], pipelineCache, json_options_);
+    FieldToJson(args["createInfoCount"], createInfoCount, json_options_);
+    FieldToJson(args["pCreateInfos"], pCreateInfos, json_options_);
+    FieldToJson(args["pAllocator"], pAllocator, json_options_);
+    HandleToJson(args["pPipelines"], pPipelines, json_options_);
+    WriteBlockEnd();
+}
```

Comment:
Please remove this, it is the same as the default generated version. We don't want custom code where generated code can work. You probably added this because the default implementation disappeared when you added `vkCreateRayTracingPipelinesKHR` to `vulkan_generators/blacklists.json`. Each generator can have its own blocklisk.blacklist JSON file: they don't have to share this one.

Either give the JSON consumer its own blocklist or remove this function from the default blocklist in the JSON consumer header and body generators with a line like `self.APICALL_BLACKLIST.remove('vkCreateRayTracingPipelinesKHR')` (untested), probably at the botton of `__init__()` or the top of `generate_feature()`.

Reply:
I've updated it accordingly, thanks.